lib/common/colors.dart

```diff
@@ -11,6 +11,7 @@ class Colors {
   static const Color _mediumGray = Color(0xFFADADAD);
   static const Color _lightWarmGray = Color(0xFFF8F6F9);
   static const Color _pixelGreen = Color(0xFF83E0BA);
+  static const Color _listPixel = Color(0xFFF8F6F9);
```

Comment:
My idea was to have these variables with pure color names (not associated to an app specific context). That's why we would need the private colors variables and the public app specific getters (with meaningful names based on the context they are used in).

Reply:
Okay, i saw this color i added is the same of `_lightWarmGray`, so i don't need to create other. =)